lib/distributed_cache.rb

```diff
@@ -3,6 +3,26 @@
 require 'message_bus/distributed_cache'
 
 class DistributedCache < MessageBus::DistributedCache
+  class << self
```

Comment:
Can we keep this change as a separate PR ?

Reply:
It is in a separate PR, but it's also required here because things get put into the caches during prefabrication which then need to be invalidated before the tests run.

Reply:
I've said that backwards. The problem is that when we jump from test to test in a group, the cache  isn't cleared, but it is unaware that the transaction was rolled back. Without prefabrication, the new would evict the old in the setup phase of each test, but now that doesn't happen.